pulp_container/app/utils.py

```diff
@@ -150,7 +150,7 @@ def determine_media_type(content_data, response):
         # translate v1 signed media_type
         if media_type == MEDIA_TYPE.MANIFEST_V1_SIGNED:
             return MEDIA_TYPE.MANIFEST_V1
-        elif media_type in (MANIFEST_MEDIA_TYPES.IMAGE or MANIFEST_MEDIA_TYPES.LIST):
+        elif media_type in MANIFEST_MEDIA_TYPES.IMAGE or media_type in MANIFEST_MEDIA_TYPES.LIST:
             return media_type
```

Comment:
does this need a data repair command/migration?

Reply:
We were basically skipping the fixed `elif` condition and passing the evaluation to https://github.com/pulp/pulp_container/blob/0b979a73676116ae959d6a16e01d61616b1730ac/pulp_container/app/utils.py#L164-L170. I think we are fine.